allensdk/brain_observatory/behavior/trials_processing.py

```diff
@@ -90,224 +77,3 @@
     'localhost': 'localhost'
 }
 RIG_NAME = {k.lower(): v for k, v in RIG_NAME.items()}
-
-
-def calculate_reward_rate(response_latency=None,
```

Comment:
I guess it wasn't really necessary, the main reason was that I wanted the code to all be in the same place rather than being split up in the Trials class and trial_processing module